docs/spec/spec.md

```diff
@@ -314,69 +280,55 @@ metadata:
   selfLink: ...
   ...
 
-# spec contains one of several possible rollout styles
-spec:  # One of "runLatest", "release", "pinned" (DEPRECATED), or "manual"
-
-  # Example, only one of "runLatest", "release", "pinned" (DEPRECATED), or "manual" can be set in practice.
-  runLatest:
-    configuration:  # serving.knative.dev/v1alpha1.ConfigurationSpec
-      # +optional. The build resource to instantiate to produce the container.
-      build: ...
-      revisionTemplate: ... # Deprecated way of expressing the same as template.
-      template:
-        metadata:
-          # Name may be specified here following the same rules as Configuration.
-          # When the name collides, the Route must not be updated to the
-          # conflicting Revision.
-        spec: # serving.knative.dev/v1alpha1.RevisionSpec
-          container: ... # See the Container section below
-          volumes: ... # Optional
-          containerConcurrency: ... # Optional
-          timeoutSeconds: ... # Optional
-          serviceAccountName: ...  # Name of the service account the code should run as
-
-  # Example, only one of "runLatest", "release", "pinned" (DEPRECATED), or "manual" can be set in practice.
-  pinned:
-    revisionName: myservice-00013  # Auto-generated revision name
-    configuration:  # serving.knative.dev/v1alpha1.ConfigurationSpec
-      # +optional. The build resource to instantiate to produce the container.
-      build: ...
-      revisionTemplate: ... # Deprecated way of expressing the same as template.
-      template:
-        spec: # serving.knative.dev/v1alpha1.RevisionSpec
-          container: ... # See the Container section below
-          volumes: ... # Optional
-          containerConcurrency: ... # Optional
-          timeoutSeconds: ... # Optional
-          serviceAccountName: ...  # Name of the service account the code should run as
-
-  # Example, only one of "runLatest", "release", "pinned" (DEPRECATED), or "manual" can be set in practice.
-  release:
+spec:
+  template:
+    metadata:
+      # Name may be specified here following the same rules as Configuration.
+      # When the name collides, the Route must not be updated to the
+      # conflicting Revision.
+    spec:
+      # Service supports the full inline Configuration spec.
+      containers: ... # See the Container section below
+      volumes: ...
+      serviceAccountName: ...
+      containerConcurrency: ...
+      timeoutSeconds: ...
```

Comment:
We use "..." for an integer value here, but an example of "100" for the percent field below. There are a couple of other places as well where we have inconsistencies in the value field.

We should be consistent across this document. Something like:

* Acceptable ranges for integers [0-99]
* Example (i.e. "myservice-00013") for strings
* "..." for structures
* "true | false" for boolean

Reply:
I change this to `NNN`